plugin/storage/es/spanstore/writer.go

```diff
@@ -101,12 +101,15 @@ func NewSpanWriter(p SpanWriterParams) *SpanWriter {
 }
 
 // CreateTemplates creates index templates.
-func (s *SpanWriter) CreateTemplates(spanTemplate, serviceTemplate string) error {
-	_, err := s.client.CreateTemplate("jaeger-span").Body(spanTemplate).Do(context.Background())
+func (s *SpanWriter) CreateTemplates(spanTemplate, serviceTemplate, indexPrefix string) error {
+	if indexPrefix != "" {
+		indexPrefix += "-"
+	}
+	_, err := s.client.CreateTemplate(indexPrefix + "jaeger-span").Body(spanTemplate).Do(context.Background())
 	if err != nil {
 		return err
 	}
-	_, err = s.client.CreateTemplate("jaeger-service").Body(serviceTemplate).Do(context.Background())
+	_, err = s.client.CreateTemplate(indexPrefix + "jaeger-service").Body(serviceTemplate).Do(context.Background())
```

Comment:
are we adding new functionality here to support custom index prefixes that was not available previously? It is not mentioned in the PR title/description

Reply:
This was done to maintain support for having same ES backend for multiple Jaeger implementations. More about this in this comment -

https://github.com/jaegertracing/jaeger/pull/2454#issuecomment-741933927

Reply:
that's great, I am just saying we need to call this out in the PR title/description/changelog

Reply:
Yep, agreed- will add to description of PR.

Reply:
Added it in description